algorithms/branching.py

```diff
@@ -1,11 +1,11 @@
 from supporting_components.graph import Graph, Vertex
-from algorithms.color_refinement import color_refinement, get_colors
+from algorithms.color_refinement import get_colors
 from algorithms.decide_gi import is_balanced_or_bijected
 from typing import List, Dict
 from math import inf
 
 
-def count_isomorphisms(G: 'Graph', D: 'List[Vertex]', I: 'List[Vertex]', count_flag: 'Bool'):
+def count_isomorphisms(G: 'Graph', D: 'List[Vertex]', I: 'List[Vertex]', count_flag: 'Bool', color_refinement):
```

Comment:
Is color refinement de functie die je meegeeft, dus of de normale of de snelle? Ik weet niet of hier dan nog een type aan gehangen kan worden.

Reply:
Wordt dan ook door hier dat ding precies color_refinement te noemen, hieronder op regel 21 color_refinement uitgevoerd op de graaf met de gespecificeerde methode?

Reply:
Type van dit argument is volgens type() <class 'function'>.
De type function toevoegen is overzichtelijker anders lijkt het net een import.

Reply:
Ja, omdat color_refinement zelf niet is geimporteerd uit color_refinement algoritme. Maar ik zal hem een duidelijkere naam geven